scripts/benchall.py

```
@@ -0,0 +1,136 @@
+        #    'manual_column_names': True,
+        #    'keys': ['t.\\$date'],
+        #    'additional_order_by': [],
+        #    }),
+        #(clickhouse_native_json_bench, {
+        #    'manual_column_names': False,
+        #    'keys': [],
+        #    'additional_order_by': [],
+        #    'timestamp_key': True
+        #    }),
+        (clp_presto_bench, {
+            'dataset_variation': "cleaned_log"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)